### ISSUE TITLE
[Feature Request] Allow Frames insertion without Canvas Fill

### ISSUE THREAD

**Issue by xiaoas** [NickeManarin/ScreenToGif#1007]
In gif editor, when Inserting a Image into the sequences, a non-transparent canvas is required behind the image(which has a alpha channel), making the frame none-transparent.

Desired Behaviour: The "Canvas Fill" could be toggled off. Or the canvas fill color could be specified as transparent.

**Comment by NickeManarin** [comment]
Yeah, that's something planned, along with the editor support for layers.

**Comment by byzod** [comment]
The more I use stg, the more how important I found layers is

Currently all works related to layers I had to use photoshop instead, it's usable but not comfortable, I prefer stg if possible
